components/TaggedValue/TaggedValue.jsx

```diff
@@ -0,0 +1,30 @@
+require('./TaggedValue.scss')
+
+import React from 'react'
+import classNames from 'classnames'
+
+const TaggedValue = ({title,subText,style,count}) => {  
+  const taggedValueClass = classNames(
+    'TaggedValue',
+    { [`${style}`]: style,
```

Comment:
Let me rephrase it again. What I am saying is that the css rule where you are setting the background color of the label is `tagged-value-bg` right now and you are providing the custom class at component level i.e. parent of the `tagged-value-bg`. Now, if you set a default value, lets say `default`, to the `style` property, you can remove the css rule of background color from `tagged-value-bg` class and instead use following:
`
.TaggedValue {
  &.default {
    .tagged-value-bg {
      background-color: $accent;
    }
  }
}
`
Does this makes sense?